src/test/regress/base_schedule

```diff
@@ -5,3 +5,4 @@ test: multi_cluster_management
 test: multi_test_helpers
 test: multi_create_table multi_behavioral_analytics_create_table
 test: multi_load_data
+test: multi_create_fdw
```

Comment:
can this be parallelised with another test? (tests on the same line run in parallel)

Reply:
`multi_create_fdw` starts with `SET citus.next_shard_id TO 390000;` but otherwise shouldn't conflict with anything